components/lib/column/column.d.ts

```diff
@@ -963,6 +963,12 @@ export interface ColumnProps {
      * @defaultValue false
      */
     frozen?: boolean | undefined;
+    /**
+     * When enabled and cellEditorValidator is set, force to call cellEditorValidator
+     * before cell editor is closed. If cellEditorValidator returns false, editor stays open.
+     * @defaultValue false
+     */
+    forceHasValidValue?: boolean | undefined;
```

Comment:
Is this property really needed as the check for `cellEditValidator` already exists so if you have a cellEditValidator defined it should check right?  Or is this a case where you have a `cellEditValidator` defined but DONT want to check it?

Reply:
I guess I don't understand why you would declare a `cellEditValidator` and then want to skip using it? Just don't delcare a `cellEditValidator` right?

Reply:
Well this is just for the case that submit is true and you want to force that the cell contains a valid value. Since submit is false only when cancelling edit cell on Escape key and cellValidator only fires in that case, I just needed to add some logic to force call cellValidator even if the cell value was submited.

https://github.com/primefaces/primereact/blob/133e2d02559e9eccfe96b1e55a5489071ea90c10/components/lib/datatable/BodyCell.js#L181-L182

In my case, I have the autocomplete component which must have a valid value (i.e, the value must be an object picked from the list) and when the switchCellToView method is called I need to force check that the value is correct to properly avoid close the cell but without that logic I cannot do that because as I said, cellValidator only is being executed when submit value is false which inly happens on cell edit cancel.

Reply:
How about `cellEditValidateOnClose` ?

Reply:
Sounds good to me. I'll change now